server/views/partials/new_workAndSkillsPage/README.md

```diff
@@ -0,0 +1,5 @@
+Template partials for the new Work & Skills page introduced in epic RR-575
+The new W&S view is only rendered if the NEW_WORK_AND_SKILLS_TAB_ENABLED feature toggle is enabled
+
+TODO - The feature toggle, the "new_" prefix, and this README will be removed in RR-607
+
```

Comment:
This new directory `partials/new_workAndSkillsPage` is a verbatim copy of the existing `partials/workAndSkillsPage` directory, with the addition of this `README`